docs/getting-started/config/authentication.md

```diff
@@ -8,43 +8,48 @@ Audience: Admins or Developers
 
```

Comment:
Outside of the diff, but can we change the description to `How to setup authentication for your Backstage app`. Both grammar and being consistent about Backstage app as the a single instance of Backstage customized for your company.

Reply:
Udpated